hwdb/60-keyboard.hwdb

```diff
@@ -492,9 +492,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHP440G2:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPProBook445G1NotebookPC:pvr*
 # HP ProBook 450 G0
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPProBook450G0:pvr*
+ KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
+
```

Comment:
nope, it's just that the diff algorithm is stupid in this case. I just moved all the `KEYBOARD_KEY_` on a separate section for the ProBook. The `+` should be on line 499, not on 495, but diff is messing this up.